### ISSUE TITLE
no-spaced-func complains about multi-line hash parameter

### ISSUE THREAD

**Issue by rosskevin** [eslint/eslint#4579]
## Env
```
grunt-eslint: 17.3.1
eslint: 1.10.1 
```

## Error
```
130:39  error    Unexpected space between function name and paren            no-spaced-func
```

## Code
```javascript
      let $container = $("<div></div>") // mock
      let $rippleElement = $("<div></div>") 
      let relX = 1
      let relY = 2
      let rippleColor = 'red'

      $rippleElement.addClass("ripple").css({
        "left": relX,
        "top": relY,
        "background-color": rippleColor
      })

      /**
       * Append the ripple to the wrapper
       */
      $container.append($rippleElement)  // line 130
```

## Summary

The line 130 problem report may be a mistake, when I comment it out it complains about the block above that is passing a hash as a parameter to `#css()`.   But then again, I converted the `#css()` block to interpolation, and it still complains about line 130.

It doesn't appear to me that this code violates [the rule](http://eslint.org/docs/rules/no-spaced-func.html), but then again this is my first pass on an es6 conversion.

## Expectation

I could be wrong, but I don't think the code provided is semantically incorrect.

**Comment by eslintbot** [comment]
@rosskevin Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by nzakas** [comment]
I'm unable to reproduce this problem locally. Can you try to narrow things down some more? Is there anything on line 131?

**Comment by rosskevin** [comment]
Hello @nzakas - I tried to provide a small sample above but perhaps the other errors are leading me down the wrong path.  I've fixed all the remaining, and indeed there is a line following which is problematic.  line 130 is now 108.
```
/bootstrap-material-design/js/src/ripples.js
  108:44  error  Unexpected space between function name and paren            no-spaced-func
  111:7   error  Unexpected newline between function and ( of function call  no-unexpected-multiline
  111:16  error  Statement inside of curly braces should be on next line     brace-style
```

These are my remaining issues, the source can be viewed here: 
https://github.com/FezVrasta/bootstrap-material-design/blob/v4-dev/js/src/ripples.js

**Comment by platinumazure** [comment]
What about your ESLint config? Can you include that?

**Comment by rosskevin** [comment]
Eslint config:
https://github.com/FezVrasta/bootstrap-material-design/blob/v4-dev/js/.eslintrc

**Comment by rosskevin** [comment]
So this is the culprit:

```javascript
      // Make sure the ripple has the styles applied (ugly hack but it works)
      (() => { return window.getComputedStyle(this._rippleElement[0]).opacity })()
```

**Comment by rosskevin** [comment]
I'm not sure what that code was about, but I've rewritten it as a private method and all is well now.  Thank you.
